samples/sample_tap_hostile/hostile_streams.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import random
+import string
+from typing import Iterable
+
+from singer_sdk import typing as th
+from singer_sdk.streams import Stream
+
+
+class HostilePropertyNamesStream(Stream):
+    """
+    A stream with property names that are not compatible as unescaped identifiers
+    in common DBMS systems.
+    """
+
+    name = "hostile_property_names_stream"
+    schema = th.PropertiesList(
+        th.Property("name with spaces", th.StringType),
+        th.Property("NameIsCamelCase", th.StringType),
+        th.Property("name-with-dashes", th.StringType),
+        th.Property("Name-with-Dashes-and-Mixed-cases", th.StringType),
+        th.Property("5name_starts_with_number", th.StringType),
```

Comment:
See my comment above regarding adding a prefix instead of removing leading digits

```suggestion
        th.Property("5name_starts_with_number", th.StringType),
        th.Property("6name_starts_with_number", th.StringType),
        th.Property("7name_starts_with_number", th.StringType),
```